bedrock/firefox/templates/firefox/new/basic/base_download.html

```diff
@@ -59,14 +60,13 @@
   <div class="mzp-c-notification-bar mzp-t-warning" data-nosnippet="true">{{ ftl('firefox-new-mozilla-no-longer-provides', fallback='firefox-new-youre-using-an-insecure-outdated', url='https://support.mozilla.org/kb/end-support-windows-xp-and-vista') }}</div>
   <![endif]-->
 
-  {% call hero(
-    title=hero_title,
-    class='mzp-has-image hero-firefox t-narrow',
-    include_cta=True,
-    image_url=hero_img,
-    heading_level=1
-  ) %}
+  {% call split(
+    block_class='mzp-l-split-center-on-sm-md',
+    image=resp_img(hero_img)
 
```

Comment:
**issue (non-blocking)**: local doesn't allow image to overflow off side of screen, image gets a bit small
<img width="704" alt="Screen Shot 2023-01-13 at 3 40 05 PM" src="https://user-images.githubusercontent.com/19650432/212359861-d6869b64-83ee-4745-b453-c4f4b14bd488.png">

vs prod
<img width="724" alt="Screen Shot 2023-01-13 at 3 40 26 PM" src="https://user-images.githubusercontent.com/19650432/212360007-19521dc7-1ed0-4f10-a197-29f79fddf3cd.png">

**suggestion**: use `mzp-l-split-media-overflow` to keep current prod styling